tutorials/setup.md

```diff
@@ -69,7 +69,7 @@ apt-get update -qq && apt-get install -y \
 
 If using an older release or a long-term support release, be careful to double-check that the version of `runc` is new enough (running `runc --version` should produce `spec: 1.0.0`), or else build your own.
 
-Be careful to double-check that the version of golang is new enough, version 1.12.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
+Be careful to double-check that the version of golang is new enough, version 1.10.x or higher is required.  If needed, golang kits are available at https://golang.org/dl/
```

Comment:
If we really want to support go 1.10 then I guess we also have to support 1.11. For that we would have to add (at least) build targets to our CI which still test the older versions.

Reply:
I'm OK with go1.10 and go1.12 minikube, but I suppose other people might run 1.11 or 1.13